app/views/user/index.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>User</h1>
-<p>Here is the list of user for the blog app</p>
+<% if @users.blank? %>
+<p>There are not books</p>
+<% else %>
+<p>These are current Users.</p>
+
+<ul>
+<% @users.each do |u| %>
+<li>
+ <img src="<%= u.photo %>" alt="image">
+ <h2><%= link_to u.name, {:action => 'show', :id => u.id}  %></h2>
+ <p>Number of posts: <%= u.posts.length %> </p>
+  </li>
+<% end %>
+</ul>
+<button>See all the post</button>
+
```

Comment:
- [ ] This button should be on the User's **_show_** page and not the User's _**index**_ page based on the wireframe provided. The **_See all posts_** button on the user's _**show**_ page should link to the page with all the `posts` for a single user based on the wireframe. This means it should link to the **_post's index page_**. Kindly transfer this button to the user's show page instead of this page. 👍 😎 

### Wireframe

![image](https://user-images.githubusercontent.com/77106962/224070983-77993eaf-eaa2-4035-93d4-d37c3b255785.png)